src/components/Global/Search.vue

```diff
@@ -50,6 +51,10 @@ export default {
         return this.$t('global.form.search');
       },
     },
+    isSearchDisabled: {
+      type: Boolean,
+      default: true,
```

Comment:
Give default value as **false** instead. If we give as true, the search will be disabled by default if we give as true. Other locations we are not trying to disable. Might cause problems. We will only disable when we want.